render/render.go

```diff
@@ -79,35 +87,96 @@ func (c *Canvas) DrawRect(x, y, width, height int, col color.RGBA, thickness int
 	c.FillRect(x+width-thickness, y, thickness, height, col)
 }
 
+// FontStyle represents text rendering options.
+// CSS 2.1 §16 Text and §15 Fonts
+type FontStyle struct {
+	Size           float64 // Font size in pixels
+	Weight         string  // Font weight: "normal" or "bold"
+	Style          string  // Font style: "normal" or "italic"
+	Decoration     string  // Text decoration: "none" or "underline"
+}
+
 // DrawText draws text at the given position with the given color.
 // CSS 2.1 §16 Text
 func (c *Canvas) DrawText(text string, x, y int, col color.RGBA) {
-	// Use basicfont.Face7x13 as a simple built-in font
-	face := basicfont.Face7x13
-
-	// Calculate the bounding box for the text
-	width := len(text) * face.Advance
-	height := face.Height
-
-	// Create a temporary image for just the text
-	textImg := image.NewRGBA(image.Rect(0, 0, width, height))
+	// Use default font style for backward compatibility
```

Comment:
Don’t hardcode default font style here - can it refer to a default style sheet or similar?